roles/common/tasks/netdata.yml

```diff
@@ -0,0 +1,65 @@
+---
+# install netdata on node
+- name: Download the installation script
+  get_url:
+    url: https://my-netdata.io/kickstart.sh
+    dest: ~/kickstart.sh
```

Comment:
Can this go to /tmp instead?

Reply:
It could, but the kickstart script gets removed  during the cleanup script.  I'll make the change though